bindata/assets/alerts/kube-apiserver-slos-basic.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       labels:
         long: 1h
         namespace: openshift-kube-apiserver
-        severity: critical
+        severity: warning
```

Comment:
I am on the fence regarding both of the severity changes of the KubeAPIErrorBudgetBurn alerts. Although these alerts are hardly actionable, they are usually symptoms of things going very wrong in the cluster. Most of the time, these availability issues will propagate all over the cluster since a lot of applications depend on the apiserver. As such, I still think that customers would want to be informed right away when such a thing happens, and we should give them the keys to solve this kind of issue, even though it might need further investigation from their side.

It is also worth mentioning that some OSD folks wrote this runbook to help diagnose and mitigate the issues reported by these alerts: https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-apiserver-operator/KubeAPIErrorBudgetBurn.md. If we think that it is lacking we could maybe improve it? wdyt @tkashem?